consumer-services/src/main/scala/za/co/absa/spline/consumer/service/repo/ExecutionEventRepository.scala

```diff
@@ -15,27 +15,20 @@
  */
 package za.co.absa.spline.consumer.service.repo
 
-import za.co.absa.spline.consumer.service.model.{ExecutionEvent, PageRequest, PageableExecutionEventsResponse, SortRequest}
+import za.co.absa.spline.consumer.service.model.{PageRequest, PageableExecutionEventsResponse, SortRequest}
 
 import scala.concurrent.{ExecutionContext, Future}
 
 trait ExecutionEventRepository {
 
-  def findByTimestampRange
-  (
+  def findByTimestampRange(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The number of parameters should not exceed 8](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4536902)